webextensions/api/action.json

```diff
@@ -425,14 +425,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action/openPopup",
             "support": {
               "chrome": {
-                "version_added": "88",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "#extension-apis",
-                    "value_to_set": "Enabled"
-                  }
-                ]
+                "version_added": "118"
```

Comment:
The API is still not available by default, so it should still be marked as experimental: https://github.com/mdn/browser-compat-data/issues/16442#issuecomment-1957200999

Reply:
@Rob--W according to the [schema](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#status-information), and the linter, you can only mark an API as experimental at the API level, not the browser. Therefore, added a note instead.